src/themes/dark.scss

```diff
@@ -33,7 +33,8 @@
   --wxc-avatar--placeholder-9--background: #{$wxc-slate-70};
   --wxc-avatar--placeholder-10--background: #{$wxc-violet-70};
   --wxc-avatar--placeholder-11--background: #{$wxc-purple-70};
-  --wxc-avatar--bot-status-color: #{$wxc-blue-60};
+  --wxc-avatar--bot-status-color: #{$wxc-gray-60};
```

Comment:
Should this be changed?

Reply:
I resolved some conflicts and I kept here the wrong version for this change. I modified it now back.